www/components/itinerum_tutorial/Tutorial.jsx

```diff
@@ -1,11 +1,19 @@
 /*
  * src/app/help-panel/Tutorial.jsx
  */
-
 class TutorialComponent extends React.Component {
+    propTypes: {
+        test: React.PropTypes.string,
+        isValid: React.PropTypes.bool,
+        schema: React.PropTypes.object,
+        result: React.PropTypes.object
+    }
     render() {
         return(
                 <div className="content-wrapper">
+                    <div>
+                        This is the test -> {this.props.test}
+                    </div>
```

Comment:
Hopefully something like this works.

```suggestion
                    <div>
                        This is the first name -> {this.props.schema.firstname}
                        This is the last name -> {this.props.schema.lastname}
                    </div>
```

Reply:
Awesome! This works perfectly. (Surprised it is so simple.)